src/components/filters/DelivaryTime.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchRestaurants } from '../../actions';
+import styles from '../Filters.module.sass'
+
+class DelivaryTime extends React.Component {
+    changeDeliveryTimeFilter = (e) => {
+        const params = { max_delivery_time: e.target.value};
+
+        this.props.fetchRestaurants(params);
+    };
+
+    render() {
+        return (
+            <div>
+                <h5 className="ui header">Speed:</h5>
+                <select className={`ui dropdown ${styles['filterBox']}`} onChange={this.changeDeliveryTimeFilter} value={this.props.restaurants.filters['max_delivery_time']} >
+                    <option value=''>Speed</option>
+                    <option value='15'>15</option>
```

Comment:
Could fill an array and `map` through the values to render each `<option>` in one liner